test/ConcreteOperations/convex_hull.jl

```diff
@@ -185,4 +185,9 @@ for N in [Float64, Rational{Int}]
     # binary convex hull with an empty set is identical to unary
     @test convex_hull(S, EmptySet{N}(1)) == convex_hull(EmptySet{N}(1), S) == S
     @test convex_hull(P, EmptySet{N}(2)) == convex_hull(EmptySet{N}(2), P) == Pc
+
+    v = [[-0.0, 2], [2.0, 0], [1.0, 0], [0.0, 1], [0.0, 1.5]]
+    v_ch = [[0.0, 1.0], [1.0, 0.0], [2.0, 0.0], [-0.0, 2.0]]
+
```

Comment:
The numeric type `N` should be used instead of just `Float64` in the tests.
```suggestion
    # see #3455
    v = [N[-0.0, 2], N[0, 0], N[1, 0], N[0, 1], N[0, 3//2]]
    v_ch = [N[0, 1], N[1, 0], N[2, 0], N[-0.0, 2]]
```